source/ShipyardPanel.cpp

```diff
@@ -367,22 +370,25 @@ void ShipyardPanel::BuyShip(const string &name)
 
 void ShipyardPanel::SellShipAndOutfits()
 {
+	// false, as in: sell the outfits, don't put them in storage
 	SellShip(false);
 }
 
 
 
 void ShipyardPanel::SellShipChassis()
 {
+	// true, as in: keep the outfits in storage, only sell the ship
 	SellShip(true);
 }
 
 
 
-void ShipyardPanel::SellShip(bool toStorage)
+// ReSharper disable once CppDFAUnreachableFunctionCall
```

Comment:
This comment doesn't make sense to me. Could you clarify it a bit?

Reply:
It should be removed. If your IDE doesn't like the code, it's a problem with the IDE.

Reply:
One of the nice things about the IDE I was using, JetBrains CLion, was that it was very helpful with it's suggestions about issues with the code.  However, in this case, all paths to SellShip go through a Dialog.
void ShipyardPanel::Sell(bool storeOutfits)
```c++
		GetUI()->Push(new Dialog(this, &ShipyardPanel::SellShipChassis, message, Truncate::MIDDLE));
	}
	else
		GetUI()->Push(new Dialog(this, &ShipyardPanel::SellShipAndOutfits, message, Truncate::MIDDLE));
``` 
to 
void ShipyardPanel::SellShipAndOutfits() or void ShipyardPanel::SellShipChassis()
to
void ShipyardPanel::SellShip(bool storeOutfits)

And I wanted it to be quiet about the error, so I used the menu system to insert the comment that would make it shut up:
![image](https://github.com/user-attachments/assets/e6bd0dc8-d5f2-424d-a66b-7f6a15c9c69d)

I can remove it if it's problematic.

Reply:
If there's a problem in the code, feel free to fix it (but maybe not in this PR if it's not related). If it's just the IDE complaining, but the code is fine, please don't add IDE-specific suppression comments.